src/views/Home.vue

```diff
@@ -1,83 +1,88 @@
 <template>
-<span style="scroll">
-	<div class="fade">
-		<nav-bar/>
-		<hero/>
-	</div>
-	<!-- Timeline -->
-	<about/>
-	<!-- Tab -->
-	<speakers/>
-	<!-- Carousel -->
-	<testimonials/>
+  <span style="scroll">
+    <div class="fade">
+      <nav-bar />
+      <hero />
+    </div>
+    <!-- Timeline -->
+    <about />
+    <!-- Tab -->
+    <speakers />
+    <!-- Carousel -->
+    <testimonials />
 
-	<div id="sponsors">
-		<h2 class="sponsor-headline">Our 2020 Sponsors and Partners</h2>
-		<img
-			src="../assets/sponsors2020.svg" 
-			alt="Flik, Geotab, lleTTonna, Flipp, FoundersBeta, RBC, jack.org, JobJunxion" class="sponsor-img"
-		>
-	</div>
+    <div id="sponsors">
+      <h2 class="sponsor-headline">Our 2020 Sponsors and Partners</h2>
+      <img
+        src="../assets/sponsors2020.svg"
+        alt="Flik, Geotab, lleTTonna, Flipp, FoundersBeta, RBC, jack.org, JobJunxion"
+        class="sponsor-img"
+      />
+    </div>
 
-	<!-- Dropdown -->
-	<faq/>
-	<!-- Unsure? -->
-	<team/>
+    <!-- Dropdown -->
+    <faq />
+    <!-- Unsure? -->
+    <team />
 
-	<div class="footer">
-		<a href="https://www.facebook.com/cutc2021/" target="_blank">Facebook</a>
-		<a href="https://www.instagram.com/cutc2021/" target="_blank">Instagram</a>
-		<a href="https://www.linkedin.com/company/canadian-undergraduate-technology-conference/" target="_blank">LinkedIn</a>
-	</div>
-</span>
+    <bottom />
+  </span>
 </template>
 
 <script>
-import NavBar from '../components/NavBar.vue';
-import Hero from '../components/homePage/Hero.vue';
-import About from '../components/homePage/About.vue';
-import Speakers from '../components/homePage/Speakers.vue';
-import Testimonials from '../components/homePage/Testimonials.vue';
-import Faq from '../components/homePage/Faq.vue';
-import Team from '../components/homePage/Team.vue';
-
+import NavBar from "../components/NavBar.vue";
```

Comment:
In general, try to keep commit changes as minimal as possible, so avoid reindenting/requoting a file unless necessary